pulpcore/pulpcore/app/manage.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 import os
 import sys
+from dynaconf.contrib import django_dynaconf  # noqa
```

Comment:
Maybe this line should go after `os.environ.setdefault("DJANGO_SETTINGS_MODULE", ...`